VideoLocker/src/main/java/org/edx/mobile/task/GetCommentListTask.java

```diff
@@ -10,17 +10,25 @@
 
     String threadId;
     int page = 1;
+    boolean isQuestionType;
+    boolean shouldGetEndorsed;
 
-    public GetCommentListTask(Context context, String threadId, int page) {
+    public GetCommentListTask(Context context, String threadId, int page, boolean isQuestionType,
```

Comment:
We could replace the `threadId` parameter with the `DiscussionThread` object itself, which would let us derive both the thread ID and type. Alternatively, we could create two constructors here, with and without the `shouldGetEndorsed` field, where we can derive the type from the constructor that was called. One way or another, the `isQuestionType` parameter can be removed I guess.